docs/cookbook.md

```diff
@@ -229,3 +229,28 @@ The bug here, is that inspect does not truly inspect Postgres and always returns
 the same result. `ldap2pg` will always execute the revoke query, thinking
 `mypriv` is granted to `public`, whatever the actual state of the cluster. It's
 up to you to dig in `pg_catalog.pg_database.datacl` to find SQL GRANT.
+
+
+# ldap2pg as Docker container
+
+Already familiar with Docker and willing to save the setup time you're at the right place.
+
+To run the container simply use the command:
+``` console
+$ docker run dalibo/ldap2pg --help
+```
+
+The Docker image of ldap2pg use the same configuration options as explained in the [cli](https://ldap2pg.readthedocs.io/en/latest/cli/) and [ldap2pg.yml](https://ldap2pg.readthedocs.io/en/latest/config/) sections.
+You can either export the environmnet variables with the **-e** option:
+
+``` console
+$ docker run -e PGDSN=postgres://postgres@localhost:5432/ -e LDAPURI=ldaps://localhost -e LDAPBINDDN=cn=you,dc=entreprise,dc=fr -e LDAPPASSWORD=pasglop dalibo/ldap2pg
+```
+
+ or directly mount the ldap2pg.yml configuration file.
```

Comment:
`ldap2pg.yml` is required for `sync_map`. Thus i would suggest to always mount `${PWD}/ldap2pg.yml`. The env var is an alternative to putting settings in `ldap2pg.yml` file, but `ldap2pg.yml` is still mandatory.

Reply:
It will be corrected.